client/src/views/ScorecardView.vue

```diff
@@ -2,21 +2,20 @@
   <div>
     <!-- Cover the entire page with loading element until data is ready. -->
     <div class='loading' v-if='!showScorecard'>
-      <loading :active='true' :is-full-page='false' color='#2553A0'
-               loader='bars' opacity='1' />
+      <loading :active='true' :is-full-page='false' color='#2553A0' loader='bars' opacity='1' />
     </div>
     <div v-if='showScorecard'>
       <PredictionPanel />
 
       <!-- Only display the side-panel, full-width on mobile. -->
       <ScorecardMapSearchBar class='is-hidden-mobile' />
-      <div class='columns is-variable is-centered'>
-        <div class='column'>
+      <div class='columns is-variable is-centered is-gapless'>
+        <div class='column side-panel'>
```

Comment:
do you know if this fixes the issue where the side panel shows up when theres no result: https://screenshot.googleplex.com/8h6U8AZfjy8wDoN? If not could you do that change here as well?

Reply:
Done.